client/src/components/NavBar/NavBar.tsx

```diff
@@ -1,14 +1,16 @@
-import { AppBar, Toolbar, Typography, IconButton, Grid, Button, Container } from '@material-ui/core/';
+import { AppBar, Toolbar, Typography, Grid, Button, IconButton, Container, Menu, MenuItem } from '@material-ui/core/';
+import { useState, MouseEvent } from 'react';
+import MenuDrawer from '../../components/MenuDrawer/MenuDrawer';
 import useStyles from './navBarStyles';
 import MenuIcon from '@material-ui/icons/Menu';
 import CalendarTodayOutlinedIcon from '@material-ui/icons/CalendarTodayOutlined';
 import DashboardOutlinedIcon from '@material-ui/icons/DashboardOutlined';
```

Comment:
Icons have an index file where you can clean this up `import { Menu, CalendarTodayOUtlined, DashboardOutlined } from '@material-ui/icons';`